src/nyx/environment.h

```diff
@@ -81,6 +87,9 @@
 #define VERBOSITY_ROI_INFO 4
 #define VERBOSITY_DETAILED 8
 
+// Arrow ouput
+#define ARROW_OUPUT_TYPE "--arrowOutputType"

```

Comment:
This define is declared in a wrong position in the file: it follows feature group nicknames. All the command line options are grouped together in lines 19-60.

Reply:
What's critical: the help text issued by Environment::show_cmdline_help() doesn't mention the --arrowOutputType options. You need to edit show_cmdline_help()